packages/schema-to-ts/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "zapier-schema-to-ts",
+  "version": "0.1.0",
+  "description": "Converts zapier-platform-schema JsonSchema definitions to TypeScript types",
+  "author": "Thomas Cranny <thomas.cranny@zapier.com>",
+  "license": "SEE LICENSE IN LICENSE",
+  "main": "dist/cli.js",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "clean": "rm -rf dist",
+    "prebuild": "yarn clean",
+    "build": "tsc",
+    "generate-types": "node ./dist/cli.js --schema-json ../schema/exported-schema.json --output ../core/types/zapier.generated.d.ts",
+    "generate-types:build": "yarn build && yarn generate-types",
+    "git-add": "git add ../core/types/zapier.generated.d.ts",
+    "precommit": "yarn build && yarn generate-types && yarn git-add"
+  },
+  "dependencies": {
+    "commander": "^12.1.0",
+    "deepmerge": "^4.3.1",
+    "json-schema-to-typescript": "^13.1.2",
+    "marked": "^12.0.2",
+    "pino": "^9.1.0",
+    "pino-pretty": "^11.0.0",
+    "prettier": "^3.2.5",
+    "word-wrap": "^1.2.5"
+  },
+  "devDependencies": {
+    "typescript": "^5.5.2",
+    "vitest": "^1.5.0"
```

Comment:
I had never heard of `vitest` , looks interesting! I see that is has a Jest-compatible API, so I'm curious if there was any specific reason to pick vitest over Jest? We already use Jest in other packages it may be nice to stick with Jest for standardization's sake.